### DISCUSSION TITLE
Add support for CarbonROM-cr-10.0 & 11.0

### DISCUSSION THREAD

**Discussion by Sensei-Developer** [ROM-builders/temporary#14787] [General]
PR! https://github.com/ROM-builders/temporary/pull/14786

cr-10.0 = Android 12
cr-11.0 = Android 13

@Apon77

**Comment by Apon77** [comment]
why only you need lots of rom to be supported?? please stay with few roms.

**Reply by Sensei-Developer** [reply]
because I collected this rum from the very release, because it is optimized and without gapps, all the rest are heavy for me and therefore I ask you to add this
